src/pages/components/select/style.mdx

```diff
@@ -169,7 +169,7 @@ and any proper nouns capitalized. Select text should be three words or less.
 
 <div className="image--fixed">
 
-![Structure and spacing measurements for Inline Select](images/select-style-structure-default-fluid.png)
+![Structure and spacing measurements for fluid select](images/select-style-structure-fluid.png)
```

Comment:
This image is missing the teal padding overlay on the right side for the error state.
<img width="1006" alt="Screen Shot 2023-05-10 at 3 24 21 PM" src="https://github.com/carbon-design-system/carbon-website/assets/43969356/40a554be-bd7a-4671-8f93-9f14232b8c97">